src/seccomp_notify.c

```diff
@@ -21,7 +21,7 @@
 #include <dlfcn.h>
 #include <sys/sysmacros.h>
 #include <linux/seccomp.h>
-#include <seccomp.h>
+#include <asm/unistd.h>
```

Comment:
Assuming more appropriate is to #include <syscall.h> or #include <sys/syscall.h> as seccomp_notify.c needs this to call the __NR_seccomp syscall?